docs/content/en/share_your_findings/jira_guide.md

```diff
@@ -0,0 +1,650 @@
+
+### Add a Jira Instance (Classic UI / Open-Source)
+
+1. If you have not already done so, navigate to the System Settings page and check the box on **Enable Jira Integration**. You will need to do this before the ⚙️ **Configuration \> JIRA** option shows up on the sidebar.
+​
+2. Navigate to the ⚙️ **Configuration \> JIRA**  page from the DefectDojo sidebar.
+​
+![image](images/Connect_DefectDojo_to_Jira.png)
+
+3. You will see a list of all currently configured Jira Projects which are linked to DefectDojo. To add a new Project Configuration, click the wrench icon and choose either the **Add Jira Configuration (Express)** or **Add Jira Configuration** options.
+
+#### Add Jira Configuration (Express)
+
+The Express method allows for a quicker method of linking a Project. Use the Express method if you simply want to connect a Jira Project quickly, and you aren’t dealing with a complex Jira workflow.
+
+![image](images/Connect_DefectDojo_to_Jira_2.png)
+
+1. Select a name for this Jira Configuration to use in DefectDojo. This name is simply a label for the Instance connection in DefectDojo, and does not need to be related to any Jira data.
+​
+2. Select the URL for your company’s Jira instance \- likely similar to `https://**yourcompany**.atlassian.net` if you’re using a Jira Cloud installation.
+​
+3. Enter an appropriate authetication method in the Username / Password fields for Jira:
+    * For standard **username / password Jira authentication**, enter a Jira Username and corresponding Password in these fields.
+    * For authentication with a **user's API token (Jira Cloud)** enter the Username with the corresponding **API token** in the password field.
+    * For authentication with a Jira **Personal Access Token (aka PAT, used in Jira Data Center and Jira Server only)**, enter the PAT in the password field.  Username is not used for authentication with a Jira PAT, but the field is still required in this form, so you can use a placeholder value here to identify your PAT.
+​
+4. Select the Default issue type which you want to create Issues as in Jira. The options for this are **Bug, Task, Story** and **Epic** (which are standard Jira issue types) as well as **Spike** and **Security**, which are custom issue types. If you have a different Issue Type which you want to use, please contact [support@defectdojo.com](mailto:support@defectdojo.com) for assistance.
+​
+5. Select your Issue Template, which will determine the Issue Description when Issues are created in Jira.
+
+The two types are:
+\- **Jira\_full**, which will include all Finding information in Jira Issues
+\- **Jira\_limited**, which will include a smaller amount of Finding information and metadata.
+​
+If you leave this field blank, it will default to **Jira\_full.**
+​
```

Comment:
There's some weirdness with newline/space characters here which is causing this to get all bunched up, but I'm not sure why

```suggestion
The two types are:
- **Jira\_full**, which will include all Finding information in Jira Issues
- **Jira\_limited**, which will include a smaller amount of Finding information and metadata.

If you leave this field blank, it will default to **Jira\_full.**

```